ts/src/base/Exchange.ts

```diff
@@ -2363,8 +2363,8 @@ export default class Exchange {
         let percentage = this.safeValue (ticker, 'percentage');
         let average = this.safeValue (ticker, 'average');
         let vwap = this.safeValue (ticker, 'vwap');
-        const baseVolume = this.safeValue (ticker, 'baseVolume');
-        const quoteVolume = this.safeValue (ticker, 'quoteVolume');
+        const baseVolume = this.safeString (ticker, 'baseVolume');
+        const quoteVolume = this.safeString (ticker, 'quoteVolume');
```

Comment:
to make the current PR easier to pass build, we can make a separate pr for base changes.

Reply:
these are needed here

Reply:
yes, but travis re-tests whole lib on each commit in this PR, because important-changes are being detected.
oky, as you wish.